### PR TITLE
Revert endpoint event ontologies

### PR DESCRIPTION
**Pull request by msprunck**
Design https://github.com/advthreat/iroh/pull/7167

This PR reverts the event ontologies added by #387 because they causing test failures in CTIA. We keep the new observable types as they are required by the SCA integration which has a higher priority. It will give us more time to fix the test failures in CTIA.

### CODE REVIEW

**Review by frenchy64 — APPROVED** [review]
Code looks good, could you link an issue explaining the rationale?

File: src/ctim/schemas/vocabularies.cljc [hunk 1]
```diff
@@ -259,7 +259,7 @@
     "cisco_mid"
     "cisco_uc_id"          ;; Cisco Unified Connector ID
     "cisco_cm_id"          ;; Cisco NVM UUID
-    "device"               
+    "device"
```
Comment (jyoverma) [inline src/ctim/schemas/vocabularies.cljc:262]: we need the process observables here

**Review by jyoverma — CHANGES_REQUESTED** [review]
other than adding back the observable type fields for process, the PR looks good

File: src/ctim/schemas/vocabularies.cljc (hunk 1 above)
Comment (msprunck) [reply src/ctim/schemas/vocabularies.cljc:262]: They are still there, I did not remove them.

**Comment by msprunck** [comment]
> Code looks good, could you link an issue explaining the rationale?

I updated the PR description with a link to the design doc and I explained the approach. I hope it clarifies the intent.

File: src/ctim/schemas/vocabularies.cljc (hunk 1 above)
Comment (gbuisson) [reply src/ctim/schemas/vocabularies.cljc:262]: @jyoverma I think you got confused, when you check a PR on github, the interface is showing you what changed only and not the full source files every time, hence the observables are still here, if you want to make sure select "view file" in the interface.

Comment (jyoverma) [reply src/ctim/schemas/vocabularies.cljc:262]: Ok in that case, I approve these changes

**Review by jyoverma — APPROVED** [review]
Looks good

**Pull request merged**